### PR TITLE
ENH: correct comment in autoencoder NB

### LINKED ISSUE #84: the Comments  of autoencoder example  is wrong
```python
# Building the encoder
def encoder(x):
    # Encoder Hidden layer with sigmoid activation #1
    layer_1 = tf.nn.sigmoid(tf.add(tf.matmul(x, weights['encoder_h1']),
                                   biases['encoder_b1']))
    # Decoder Hidden layer with sigmoid activation #2
    layer_2 = tf.nn.sigmoid(tf.add(tf.matmul(layer_1, weights['encoder_h2']),
                                   biases['encoder_b2']))
    return layer_2
```
> \# Decoder Hidden layer with sigmoid activation \# 2

should be:
> \# Encoder Hidden layer with sigmoid activation \# 2

and so does the decoder part.

### PR DESCRIPTION
**Pull request by Nickil21**
resolve #84

### CODE REVIEW

**Comment by aymericdamien** [comment]
Thanks for the fix!
